packages/expo-modules-core/android/src/main/java/expo/modules/kotlin/views/AutoSizingComposable.kt

```diff
@@ -15,23 +15,22 @@ enum class Direction {
 @Composable
 fun AutoSizingComposable(shadowNodeProxy: ShadowNodeProxy, axis: EnumSet<Direction> = EnumSet.allOf(Direction::class.java), content: @Composable () -> Unit) {
   Layout(
-    content = content,
     modifier = Modifier.fillMaxSize(),
-    measurePolicy = { measurables, constraints ->
-      val measurable = measurables.first()
-      val minIntrinsicWidth = measurable.minIntrinsicWidth(constraints.minHeight)
-      val minIntrinsicHeight = measurable.minIntrinsicHeight(minIntrinsicWidth)
-      val intrinsicWidth = minIntrinsicWidth.toDouble() / Resources.getSystem().displayMetrics.density
-      val intrinsicHeight = minIntrinsicHeight.toDouble() / Resources.getSystem().displayMetrics.density
-      val width: Double = if (axis.contains(Direction.HORIZONTAL)) intrinsicWidth else Double.NaN
-      val height: Double = if (axis.contains(Direction.VERTICAL)) intrinsicHeight else Double.NaN
-      shadowNodeProxy.setViewSize(width, height)
+    content = content
+  ) { measurables, constraints ->
```

Comment:
I thought putting the lambda as a named parameter within brackets would be better. The way it’s currently written implies that the lambda generates child components, which I find confusing.

Reply:
Just a syntactic preference, I've always written Layouts like this, I can revert

Reply:
Up to you ;)

Reply:
Reverted, no need to change 👍